api/src/main/java/org/openmrs/api/db/ContextDAO.java

```diff
@@ -177,4 +178,10 @@ public interface ContextDAO {
 	 * @see Context#updateSearchIndexForType(Class)
 	 */
 	public void updateSearchIndexForType(Class<?> type);
+
+	/**
+	 * @return a Connection to the OpenMRS database
```

Comment:
What do you think of mentioning that this is a pooled connection?

Reply:
Done